src/components/Footer.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+// import { Link } from 'react-router-dom';
+import mealIcon from '../images/mealIcon.svg';
+import exploreIcon from '../images/exploreIcon.svg';
+import drinkIcon from '../images/drinkIcon.svg';
+
+function Footer() {
+  return (
+    <footer
+      style={ {
+        display: 'flex',
+        justifyContent: 'space-around',
+        bottom: 0,
+        position: 'fixed',
+        width: '100%',
+        backgroundColor: 'grey',
+      } }
+      data-testid="footer"
+    >
+      <a href="/comidas">
+        <img data-testid="food-bottom-btn" src={ mealIcon } alt="food icon" />
```

Comment:
Fala Jorge, tranquilo? Cara pensei numa coisa aqui, já que nosso app vai ser pra usuários que falam português, e o atributo alt vai ser lido pelo usuário, acho que faz sentido a gente usar em português né?, por exemplo 
```suggestion
        <img data-testid="food-bottom-btn" src={ mealIcon } alt="icone de comida" />
```

Reply:
Obrigado pelo feedback! =)
Vamos observar como está vindo as informações da API. Se o conteúdo vier em português, também acho que faz sentido modificar para ficar coerente.
tamu junto!